components/usage/pkg/apiv1/billing.go

```diff
@@ -382,6 +382,61 @@ func (s *BillingService) CancelSubscription(ctx context.Context, in *v1.CancelSu
 	return &v1.CancelSubscriptionResponse{}, nil
 }
 
+func (s *BillingService) getPriceId(ctx context.Context, attributionId string) string {
+	defaultPriceId := s.stripePrices.TeamUsagePriceIDs.USD
+	attributionID, err := db.ParseAttributionID(attributionId)
+	if err != nil {
+		log.Errorf("Failed to parse attribution ID %s: %s", attributionId, err.Error())
+		return defaultPriceId
+	}
+
+	customer, err := s.GetStripeCustomer(ctx, &v1.GetStripeCustomerRequest{
+		Identifier: &v1.GetStripeCustomerRequest_AttributionId{
+			AttributionId: string(attributionID),
+		},
+	})
+	if err != nil {
+		if status.Code(err) != codes.NotFound {
+			log.Errorf("Failed to get stripe customer for attribution ID %s: %s", attributionId, err.Error())
+		}
+		return defaultPriceId
+	}
+
+	stripeCustomer, err := s.stripeClient.GetCustomer(ctx, customer.Customer.Id)
+	if err != nil {
+		log.Errorf("Failed to get customer infromation from stripe for customer ID %s: %s", customer.Customer.Id, err.Error())
+		return defaultPriceId
+	}
+
+	// if the customer has an active subscription, return that information
+	for _, subscription := range stripeCustomer.Subscriptions.Data {
+		if subscription.Status != "canceled" {
+			return subscription.Plan.ID
```

Comment:
Not familiar with the Stripe API: The `.Plan.ID` is the same as the `priceID` below? :thinking:

Reply:
yes